src/utilities.ts

```diff
@@ -547,6 +558,7 @@ export {
   logSuccess,
   getCljsReplStartCode,
   getShadowCljsReplStartCode,
+  quickPickActive as quicPickActive,
```

Comment:
Can we get rid of the `as quicPickActive` here? We'll need to fix the references to it in the tests, too.

Reply:
No idea why this coercion is here... I'll ditch it!